### ISSUE TITLE
Error when using Azure AD SSO to authenticate

### ISSUE THREAD

**Issue by davscot** [Ylianst/MeshCentral#1492]
Hi Ylian,

I made a mistake and left auto update enabled in the MeshCentral server config. As a result when trying to login to MeshCentral using Azure AD SSO I now get the following error:

`Internal Server Error`

Logs show:

```
Error: 403
    at /opt/meshcentral/node_modules/meshcentral/webserver.js:4600:38
    at Layer.handle [as handle_request] (/opt/meshcentral/node_modules/express/lib/router/layer.js:95:5)
    at next (/opt/meshcentral/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/opt/meshcentral/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/opt/meshcentral/node_modules/express/lib/router/layer.js:95:5)
    at /opt/meshcentral/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/opt/meshcentral/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/meshcentral/node_modules/express/lib/router/index.js:275:10)
    at initialize (/opt/meshcentral/node_modules/passport/lib/middleware/initialize.js:53:5)
    at Layer.handle [as handle_request] (/opt/meshcentral/node_modules/express/lib/router/layer.js:95:5)
```
**Note:** I can login to MeshCentral using local credentials so this error seems specific to Azure AD SSO only.

Unfortunately as I haven't logged into the system in a few weeks I'm unsure on which version of MeshCentral I was using when Azure SSO worked. 

**Environment specs for troubleshooting error above:**

MeshCentral ver: 0.5.62
OS running MeshCentral: Ubuntu 20.04 LTS (64bit)
NodeJS ver: 10.19.0
NPM ver: 6.14.5
MeshCentral running on built in DB.
Browser: Chrome, MS Edge

**Comment by Ylianst** [comment]
Interesting, looking at it now. In your config.json, did you specify a callback URL like this? No need to change anything, but just want to know if you config has this or not. Thanks.

```
        "azure": {
          "callbackurl": "https://server/auth-azure-callback", <-------------
          "newAccounts": true,
          "clientid": "00000000-0000-0000-0000-000000000000",
          "clientsecret": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
          "tenantid": "00000000-0000-0000-0000-000000000000"
        },
```

**Comment by Ylianst** [comment]
FYI. I can confirm, I see the same problem. Working on it now.

**Comment by davscot** [comment]
> Interesting, looking at it now. In your config.json, did you specify a callback URL like this? If not, feel free to add it and see if it works.
> 
> ```
>         "azure": {
>           "callbackurl": "https://server/auth-azure-callback", <-------------
>           "newAccounts": true,
>           "clientid": "00000000-0000-0000-0000-000000000000",
>           "clientsecret": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
>           "tenantid": "00000000-0000-0000-0000-000000000000"
>         },
> ```

Confirming my config looks the same.

**Comment by Ylianst** [comment]
Ok, hold on, new version coming out shortly with the fix for this. I am retesting all the SSO providers now.

**Comment by Ylianst** [comment]
Published MeshCentral v0.5.63 with a fix for this. Let me know if it works.

**Comment by davscot** [comment]
Perfect, working again! Thanks for your prompt response and resolution!
